services/provider/client/auth_interceptor.go

```diff
@@ -0,0 +1,41 @@
+package client
+
+import (
+	"context"
+
+	"github.com/red-hat-storage/ocs-operator/services/provider/common"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+	"k8s.io/klog"
+)
+
+type AuthInterceptor struct {
+	accessToken string
```

Comment:
Let's use a byte array with max length

Reply:
+1

Reply:
No onboarding ticket changes in this PR. This comment is not valid anymore. Marking as resolved.